fieldservice_stock_location_builder/__manifest__.py

```diff
@@ -10,7 +10,8 @@
     'website': 'https://github.com/OCA/field-service',
     'depends': [
         'fieldservice_stock',
-        'fieldservice_location_builder'
+        'fieldservice_location_builder',
+        'fieldservice_account'
```

Comment:
@osi-scampbell Why do you need `fieldservice_account`?

Reply:
That is where we moved customer_id to

Reply:
@osi-scampbell The `customer_id` field on `fsm_location` is added in `fieldservice_account_analytic`.

1. Let's remove it from here
2. In `fieldservice_account_analytic`, add a default value on the field with a method:
```
if self.parent_id:
    return self.parent_id.customer_id.id
return self.owner_id.id
```